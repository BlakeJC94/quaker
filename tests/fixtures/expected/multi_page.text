#EventID|Time|Latitude|Longitude|Depth/km|Author|Catalog|Contributor|ContributorID|MagType|Magnitude|MagAuthor|EventLocationName
ak022f7olw94|2022-11-27T16:33:06.042|57.1342|-156.35|6.4|ak|ak|ak|ak022f7olw94|ml|2.1|ak|
us6000j63n|2022-11-27T16:23:48.031|2.6479|128.0317|163.978|us|us|us|us6000j63n|mb|4.4|us|101 km N of Tobelo, Indonesia
ok2022xgel|2022-11-27T16:22:27.899|35.27066667|-98.03683333|10.16|ok|ok|ok|ok2022xgel|ml|1.59|ok|9 km WSW of Minco, Oklahoma
ok2022xgek|2022-11-27T16:22:00.860|35.29616667|-98.01166667|4.21|ok|ok|ok|ok2022xgek|ml|1.46|ok|6 km WSW of Minco, Oklahoma
ak022f7oiyw3|2022-11-27T16:19:38.222|51.8552|-177.9586|3.6|ak|ak|ak|ak022f7oiyw3|ml|1.8|ak|91 km W of Adak, Alaska
nn00852342|2022-11-27T16:15:55.365|37.1495|-117.1188|5.7|nn|nn|nn|nn00852342|ml|0.3|nn|41 km NW of Beatty, Nevada
hv73238797|2022-11-27T16:13:20.740|19.5048332214355|-155.296340942383|8.09000015258789|hv|hv|hv|hv73238797|ml|2.37|hv|9 km NW of Volcano, Hawaii
us6000j63m|2022-11-27T16:12:31.377|-2.9093|141.951|31.267|us|us|us|us6000j63m|mb|4.3|us|50 km WNW of Aitape, Papua New Guinea
ak022f7oh9n6|2022-11-27T16:11:47.060|53.5111|-165.3056|40.8|ak|ak|ak|ak022f7oh9n6|ml|2.3|ak|75 km SSE of Akutan, Alaska
nn00852122|2022-11-27T16:10:04.617|38.1683|-117.8921|8.3|nn|nn|nn|nn00852122|ml|0.6|nn|31 km SE of Mina, Nevada
nn00852119|2022-11-27T16:08:28.714|38.1675|-117.8933|8.5|nn|nn|nn|nn00852119|ml|1.4|nn|31 km SE of Mina, Nevada
av91722816|2022-11-27T15:59:13.720|51.8636666666667|-177.981333333333|6.77|av|av|av|av91722816|ml|1.29|av|92 km W of Adak, Alaska
ak022f7o3sib|2022-11-27T15:48:59.713|56.6374|-154.9353|9.7|ak|ak|ak|ak022f7o3sib|ml|1.8|ak|57 km SW of Akhiok, Alaska
ak022f7o3gi0|2022-11-27T15:47:04.112|61.6813|-151.0344|69|ak|ak|ak|ak022f7o3gi0|ml|1.6|ak|31 km WNW of Susitna, Alaska
ci40381560|2022-11-27T15:43:38.000|33.4701667|-116.5661667|8.7|ci|ci|ci|ci40381560|ml|0.54|ci|14km SE of Anza, CA
av91722806|2022-11-27T15:40:50.960|51.8491666666667|-177.983666666667|-1.62|av|av|av|av91722806|ml|1.13|av|93 km W of Adak, Alaska
av91722801|2022-11-27T15:39:47.330|51.8448333333333|-178.003666666667|-1.73|av|av|av|av91722801|ml|1.09|av|Andreanof Islands, Aleutian Islands, Alaska
av91078048|2022-11-27T15:30:25.710|51.8533333333333|-177.976|4.53|av|av|av|av91078048|ml|1.12|av|Andreanof Islands, Aleutian Islands, Alaska
us7000isrm|2022-11-27T15:29:38.281|-34.692|-71.8408|43.754|us|us|us|us7000isrm|mb|4.2|us|43 km W of Santa Cruz, Chile
ok2022xgcr|2022-11-27T15:29:25.730|34.51233333|-97.2385|4.53|ok|ok|ok|ok2022xgcr|ml|1.82|ok|10 km W of Davis, Oklahoma
ak022glaqrr3|2022-12-27T16:35:22.556|55.3905|-162.0427|116.4|ak|ak|ak|ak022glaqrr3|ml|2|ak|40 km NNE of King Cove, Alaska
ak022glaolir|2022-12-27T16:25:08.675|51.2212|-178.4993|23.9|ak|ak|ak|ak022glaolir|ml|2.2|ak|148 km WSW of Adak, Alaska
ak022glanbga|2022-12-27T16:19:12.421|60.1362|-153.1622|134.8|ak|ak|ak|ak022glanbga|ml|1.9|ak|64 km E of Port Alsworth, Alaska
ak022glamjgs|2022-12-27T16:15:35.188|59.9003|-153.9665|177.8|ak|ak|ak|ak022glamjgs|ml|2.2|ak|14 km NNE of Pedro Bay, Alaska
tx2022zizf|2022-12-27T16:13:01.982|31.64076554|-104.4632122|7.314135742|tx|tx|tx|tx2022zizf|ml|2.3|tx|59 km S of Whites City, New Mexico
av91081983|2022-12-27T16:07:17.340|58.261|-155.082833333333|4.85|av|av|av|av91081983|ml|-0.37|av|85 km NNW of Karluk, Alaska
ak022glaj8pv|2022-12-27T16:00:09.955|55.6529|-157.1695|39.1|ak|ak|ak|ak022glaj8pv|ml|2.1|ak|105 km SE of Chignik, Alaska
pr71389278|2022-12-27T15:53:02.010|17.9175|-66.8796666666667|15.18|pr|pr|pr|pr71389278|md|2.28|pr|Puerto Rico region
ak022gla84q8|2022-12-27T15:48:26.063|64.8839|-147.3461|15.6|ak|ak|ak|ak022gla84q8|ml|0.8|ak|12 km NE of Badger, Alaska
ci40150615|2022-12-27T15:43:08.420|34.5485|-119.1796667|2.58|ci|ci|ci|ci40150615|ml|1.6|ci|13km NNE of Ojai, CA
ak022gla42wu|2022-12-27T15:29:37.344|61.6775|-146.5424|20.1|ak|ak|ak|ak022gla42wu|ml|1.8|ak|37 km SSE of Nelchina, Alaska
ci40150607|2022-12-27T15:28:25.360|33.2673333|-116.4221667|4.53|ci|ci|ci|ci40150607|ml|1.23|ci|5km WNW of Borrego Springs, CA
ak022gla2afl|2022-12-27T15:21:00.587|58.5761|-154.3558|10.9|ak|ak|ak|ak022gla2afl|ml|1.1|ak|99 km SSE of Kokhanok, Alaska
av91081968|2022-12-27T15:10:33.290|51.8386666666667|-178.043666666667|-0.85|av|av|av|av91081968|ml|1.27|av|
hv73284702|2022-12-27T15:06:46.690|19.2363338470459|-155.385665893555|30.5799999237061|hv|hv|hv|hv73284702|md|1.94000006|hv|10 km ENE of P?hala, Hawaii
nc73825706|2022-12-27T14:55:26.940|37.4301667|-118.5611667|3.02|nc|nc|nc|nc73825706|md|1.46|nc|4km ENE of Round Valley, CA
ci40150599|2022-12-27T14:50:47.040|34.5795|-116.8873333|5.53|ci|ci|ci|ci40150599|ml|2.99|ci|17km NNE of Lucerne Valley, CA
us7000j35l|2022-12-27T14:48:11.567|38.2753|142.0636|79.525|us|us|us|us7000j35l|mb|4.3|us|68 km ESE of Ishinomaki, Japan
ak022gl9lsdl|2022-12-27T14:44:01.692|62.8334|-150.4092|81.7|ak|ak|ak|ak022gl9lsdl|ml|1.2|ak|41 km NNE of Petersville, Alaska
ak022gl9le5s|2022-12-27T14:42:17.445|60.4051|-148.2473|15.4|ak|ak|ak|ak022gl9le5s|ml|1.5|ak|39 km NNW of Chenega, Alaska
ak0231734hzj|2023-01-26T05:29:43.518|62.1501|-148.6362|34.3|ak|ak|ak|ak0231734hzj|ml|1.2|ak|40 km NNW of Chickaloon, Alaska
ci40161943|2023-01-26T05:21:19.220|33.4838333|-116.4046667|8.71|ci|ci|ci|ci40161943|ml|0.86|ci|22km SSW of La Quinta, CA
ak023172ou6f|2023-01-26T04:56:36.558|61.8635|-149.3468|19.8|ak|ak|ak|ak023172ou6f|ml|1.5|ak|14 km NNW of Fishhook, Alaska
ci40161935|2023-01-26T04:54:04.310|34.5181667|-118.6006667|8.67|ci|ci|ci|ci40161935|ml|1.33|ci|4km NNE of Castaic, CA
ci40161927|2023-01-26T04:51:29.140|33.9033333|-118.7026667|15.68|ci|ci|ci|ci40161927|ml|1.75|ci|14km S of Malibu Beach, CA
ci40161919|2023-01-26T04:46:25.230|33.9846667|-116.6858333|10.97|ci|ci|ci|ci40161919|ml|0.61|ci|12km SW of Morongo Valley, CA
ci40161911|2023-01-26T04:43:50.800|33.4371667|-118.2303333|13.34|ci|ci|ci|ci40161911|ml|1.91|ci|14km NE of Avalon, CA
nc73837511|2023-01-26T04:36:28.760|38.8268318|-122.8251648|0.93|nc|nc|nc|nc73837511|md|1.9|nc|8km NW of The Geysers, CA
hv73313027|2023-01-26T04:32:15.710|18.9818325042725|-155.432495117188|36.5400009155273|hv|hv|hv|hv73313027|ml|2.12|hv|18 km ESE of Naalehu, Hawaii
us6000jjbm|2023-01-26T04:29:44.721|-0.0988|-77.4813|10|us|us|us|us6000jjbm|mb|4.1|us|Ecuador
us6000jjbj|2023-01-26T04:23:51.367|2.8727|126.967|35|us|us|us|us6000jjbj|mww|5.7|us|171 km NW of Tobelo, Indonesia
nc73837506|2023-01-26T04:20:14.820|38.3904991|-122.4189987|5.77|nc|nc|nc|nc73837506|md|1.6|nc|5km WSW of Yountville, CA
us6000jjbh|2023-01-26T04:16:06.492|16.6106|-25.6047|10|us|us|us|us6000jjbh|mb|5.3|us|73 km SW of Porto Novo, Cabo Verde
us6000jjbd|2023-01-26T04:07:56.356|-30.3135|-178.7238|35|us|us|us|us6000jjbd|mb|4.4|us|Kermadec Islands, New Zealand
nc73837501|2023-01-26T04:01:41.250|38.8371658|-122.8013306|2.24|nc|nc|nc|nc73837501|md|0.78|nc|7km WNW of Cobb, CA
ak0231720kmy|2023-01-26T03:43:15.465|61.8364|-150.8749|32|ak|ak|ak|ak0231720kmy|ml|1.6|ak|32 km ESE of Skwentna, Alaska
hv73313012|2023-01-26T03:35:13.550|19.1781673431396|-155.346832275391|27.8099994659424|hv|hv|hv|hv73313012|md|1.89999998|hv|14 km E of P?hala, Hawaii
us6000jjb9|2023-01-26T03:16:22.081|-1.963|-79.9953|75.211|us|us|us|us6000jjb9|mb|4.4|us|4 km SW of El Triunfo, Ecuador
pr71394478|2023-01-26T02:26:19.140|17.9728333333333|-66.8753333333333|16.03|pr|pr|pr|pr71394478|md|2.2|pr|1 km ESE of Maria Antonia, Puerto Rico
ci40161903|2023-01-26T02:23:06.320|33.1445|-116.501|11.35|ci|ci|ci|ci40161903|ml|0.89|ci|12km NE of Julian, CA
