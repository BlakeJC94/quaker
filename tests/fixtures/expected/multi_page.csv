time,latitude,longitude,depth,mag,magType,nst,gap,dmin,rms,net,id,updated,place,type,horizontalError,depthError,magError,magNst,status,locationSource,magSource
2022-11-27T16:33:06.042Z,57.1342,-156.35,6.4,2.1,ml,,,,0.38,ak,ak022f7olw94,2022-12-12T14:33:23.810Z,"",earthquake,,0.1,,,reviewed,ak,ak
2022-11-27T16:23:48.031Z,2.6479,128.0317,163.978,4.4,mb,35,110,1.979,0.62,us,us6000j63n,2022-12-01T18:10:24.040Z,"101 km N of Tobelo, Indonesia",earthquake,12.15,8.691,0.11,24,reviewed,us,us
2022-11-27T16:22:27.899Z,35.27066667,-98.03683333,10.16,1.59,ml,12,98,0.06118799283,0.17,ok,ok2022xgel,2022-11-28T15:38:12.584Z,"9 km WSW of Minco, Oklahoma",earthquake,,1.3,0.25,3,reviewed,ok,ok
2022-11-27T16:22:00.860Z,35.29616667,-98.01166667,4.21,1.46,ml,62,73,0.09358163609,0.34,ok,ok2022xgek,2022-11-28T15:38:32.841Z,"6 km WSW of Minco, Oklahoma",earthquake,,0.8,0.18,3,reviewed,ok,ok
2022-11-27T16:19:38.222Z,51.8552,-177.9586,3.6,1.8,ml,,,,0.63,ak,ak022f7oiyw3,2022-12-12T14:33:23.615Z,"91 km W of Adak, Alaska",earthquake,,0.1,,,reviewed,ak,ak
2022-11-27T16:15:55.365Z,37.1495,-117.1188,5.7,0.3,ml,8,213.74,0.189,0.1127,nn,nn00852342,2022-11-30T23:08:41.024Z,"41 km NW of Beatty, Nevada",earthquake,,5.3,1.47,6,reviewed,nn,nn
2022-11-27T16:13:20.740Z,19.5048332214355,-155.296340942383,8.09000015258789,2.37,ml,47,74,,0.209999993,hv,hv73238797,2022-11-27T16:18:51.980Z,"9 km NW of Volcano, Hawaii",earthquake,0.39,0.709999979,0.3,5,automatic,hv,hv
2022-11-27T16:12:31.377Z,-2.9093,141.951,31.267,4.3,mb,27,99,1.306,0.41,us,us6000j63m,2022-12-01T17:44:58.040Z,"50 km WNW of Aitape, Papua New Guinea",earthquake,8.44,5.453,0.142,14,reviewed,us,us
2022-11-27T16:11:47.060Z,53.5111,-165.3056,40.8,2.3,ml,,,,0.33,ak,ak022f7oh9n6,2022-12-12T14:33:30.377Z,"75 km SSE of Akutan, Alaska",earthquake,,2.5,,,reviewed,ak,ak
2022-11-27T16:10:04.617Z,38.1683,-117.8921,8.3,0.6,ml,16,160.26,0.061,0.1434,nn,nn00852122,2022-11-30T22:00:15.200Z,"31 km SE of Mina, Nevada",earthquake,,1.3,0.42,2,reviewed,nn,nn
2022-11-27T16:08:28.714Z,38.1675,-117.8933,8.5,1.4,ml,18,130.62,0.059,0.1687,nn,nn00852119,2022-11-30T21:16:55.911Z,"31 km SE of Mina, Nevada",earthquake,,1.2,0.42,4,reviewed,nn,nn
2022-11-27T15:59:13.720Z,51.8636666666667,-177.981333333333,6.77,1.29,ml,6,105,,0.06,av,av91722816,2022-11-29T04:04:52.290Z,"92 km W of Adak, Alaska",earthquake,0.33,0.42,0.196914049372555,7,reviewed,av,av
2022-11-27T15:48:59.713Z,56.6374,-154.9353,9.7,1.8,ml,,,,0.7,ak,ak022f7o3sib,2022-12-10T08:10:26.541Z,"57 km SW of Akhiok, Alaska",earthquake,,0.4,,,reviewed,ak,ak
2022-11-27T15:47:04.112Z,61.6813,-151.0344,69,1.6,ml,,,,0.53,ak,ak022f7o3gi0,2022-12-10T08:10:17.256Z,"31 km WNW of Susitna, Alaska",earthquake,,0.4,,,reviewed,ak,ak
2022-11-27T15:43:38.000Z,33.4701667,-116.5661667,8.7,0.54,ml,17,113,0.0388,0.11,ci,ci40381560,2022-11-30T20:33:17.415Z,"14km SE of Anza, CA",earthquake,0.28,0.45,0.176,14,reviewed,ci,ci
2022-11-27T15:40:50.960Z,51.8491666666667,-177.983666666667,-1.62,1.13,ml,6,109,,0.17,av,av91722806,2022-11-29T04:00:05.780Z,"93 km W of Adak, Alaska",earthquake,0.4,1.17,0.163480202382698,7,reviewed,av,av
2022-11-27T15:39:47.330Z,51.8448333333333,-178.003666666667,-1.73,1.09,ml,5,133,,0.12,av,av91722801,2022-11-29T03:52:08.750Z,"Andreanof Islands, Aleutian Islands, Alaska",earthquake,0.3,1.24,0.143509723977443,5,reviewed,av,av
2022-11-27T15:30:25.710Z,51.8533333333333,-177.976,4.53,1.12,ml,4,161,,0.03,av,av91078048,2022-12-01T00:44:21.970Z,"Andreanof Islands, Aleutian Islands, Alaska",earthquake,0.37,0.7,0.153971557884817,4,reviewed,av,av
2022-11-27T15:29:38.281Z,-34.692,-71.8408,43.754,4.2,mb,34,122,0.326,0.55,us,us7000isrm,2022-12-01T17:28:58.040Z,"43 km W of Santa Cruz, Chile",earthquake,4.73,6.897,0.146,13,reviewed,us,us
2022-11-27T15:29:25.730Z,34.51233333,-97.2385,4.53,1.82,ml,45,60,0.1097784577,0.33,ok,ok2022xgcr,2022-11-29T18:01:57.103Z,"10 km W of Davis, Oklahoma",earthquake,,1,0.23,8,reviewed,ok,ok
2022-12-27T16:35:22.556Z,55.3905,-162.0427,116.4,2,ml,,,,0.5,ak,ak022glaqrr3,2023-01-13T18:04:29.444Z,"40 km NNE of King Cove, Alaska",earthquake,,0.8,,,reviewed,ak,ak
2022-12-27T16:25:08.675Z,51.2212,-178.4993,23.9,2.2,ml,,,,0.58,ak,ak022glaolir,2023-01-13T18:04:29.254Z,"148 km WSW of Adak, Alaska",earthquake,,0.6,,,reviewed,ak,ak
2022-12-27T16:19:12.421Z,60.1362,-153.1622,134.8,1.9,ml,,,,0.53,ak,ak022glanbga,2023-01-13T18:04:24.052Z,"64 km E of Port Alsworth, Alaska",earthquake,,0.3,,,reviewed,ak,ak
2022-12-27T16:15:35.188Z,59.9003,-153.9665,177.8,2.2,ml,,,,0.64,ak,ak022glamjgs,2023-01-13T18:04:23.840Z,"14 km NNE of Pedro Bay, Alaska",earthquake,,0.4,,,reviewed,ak,ak
2022-12-27T16:13:01.982Z,31.64076554,-104.4632122,7.314135742,2.3,ml,17,68,0.03075815809,0.3,tx,tx2022zizf,2023-01-01T21:45:00.358Z,"59 km S of Whites City, New Mexico",earthquake,1.183736178,0.8776080205,0.1,26,reviewed,tx,tx
2022-12-27T16:07:17.340Z,58.261,-155.082833333333,4.85,-0.37,ml,5,103,,0.13,av,av91081983,2022-12-29T01:31:20.430Z,"85 km NNW of Karluk, Alaska",earthquake,0.45,0.71,0.271785519909245,5,reviewed,av,av
2022-12-27T16:00:09.955Z,55.6529,-157.1695,39.1,2.1,ml,,,,0.31,ak,ak022glaj8pv,2023-01-13T18:04:29.082Z,"105 km SE of Chignik, Alaska",earthquake,,7.4,,,reviewed,ak,ak
2022-12-27T15:53:02.010Z,17.9175,-66.8796666666667,15.18,2.28,md,11,228,0.05724,0.09,pr,pr71389278,2022-12-27T16:21:04.040Z,"Puerto Rico region",earthquake,0.68,0.58,0.0807175914422853,6,reviewed,pr,pr
2022-12-27T15:48:26.063Z,64.8839,-147.3461,15.6,0.8,ml,,,,0.67,ak,ak022gla84q8,2023-01-09T22:27:26.237Z,"12 km NE of Badger, Alaska",earthquake,,0.2,,,reviewed,ak,ak
2022-12-27T15:43:08.420Z,34.5485,-119.1796667,2.58,1.6,ml,44,37,0.08652,0.23,ci,ci40150615,2023-01-25T01:14:19.070Z,"13km NNE of Ojai, CA",earthquake,0.18,0.46,0.15,23,reviewed,ci,ci
2022-12-27T15:29:37.344Z,61.6775,-146.5424,20.1,1.8,ml,,,,0.63,ak,ak022gla42wu,2023-01-09T22:27:26.002Z,"37 km SSE of Nelchina, Alaska",earthquake,,0.7,,,reviewed,ak,ak
2022-12-27T15:28:25.360Z,33.2673333,-116.4221667,4.53,1.23,ml,30,50,0.004283,0.19,ci,ci40150607,2022-12-27T16:04:12.835Z,"5km WNW of Borrego Springs, CA",earthquake,0.29,0.69,0.17,27,reviewed,ci,ci
2022-12-27T15:21:00.587Z,58.5761,-154.3558,10.9,1.1,ml,,,,0.42,ak,ak022gla2afl,2023-01-20T18:57:14.153Z,"99 km SSE of Kokhanok, Alaska",earthquake,,0.5,,,reviewed,ak,ak
2022-12-27T15:10:33.290Z,51.8386666666667,-178.043666666667,-0.85,1.27,ml,5,135,,0.21,av,av91081968,2022-12-29T01:14:42.660Z,"",earthquake,0.72,3.46,0.316800895387544,5,reviewed,av,av
2022-12-27T15:06:46.690Z,19.2363338470459,-155.385665893555,30.5799999237061,1.94000006,md,40,144,,0.150000006,hv,hv73284702,2022-12-27T15:09:54.610Z,"10 km ENE of P?hala, Hawaii",earthquake,0.69,0.819999993,1.35000002,10,automatic,hv,hv
2022-12-27T14:55:26.940Z,37.4301667,-118.5611667,3.02,1.46,md,14,154,0.1824,0.06,nc,nc73825706,2022-12-28T20:45:14.690Z,"4km ENE of Round Valley, CA",earthquake,0.39,3.01,0.066,11,reviewed,nc,nc
2022-12-27T14:50:47.040Z,34.5795,-116.8873333,5.53,2.99,ml,77,29,0.1383,0.14,ci,ci40150599,2023-01-03T20:19:21.040Z,"17km NNE of Lucerne Valley, CA",earthquake,0.11,0.61,0.121,153,reviewed,ci,ci
2022-12-27T14:48:11.567Z,38.2753,142.0636,79.525,4.3,mb,21,176,2.627,0.54,us,us7000j35l,2023-01-09T22:42:02.040Z,"68 km ESE of Ishinomaki, Japan",earthquake,9.75,11.295,0.306,3,reviewed,us,us
2022-12-27T14:44:01.692Z,62.8334,-150.4092,81.7,1.2,ml,,,,0.46,ak,ak022gl9lsdl,2023-01-09T22:27:29.760Z,"41 km NNE of Petersville, Alaska",earthquake,,0.9,,,reviewed,ak,ak
2022-12-27T14:42:17.445Z,60.4051,-148.2473,15.4,1.5,ml,,,,0.94,ak,ak022gl9le5s,2023-01-09T22:27:29.570Z,"39 km NNW of Chenega, Alaska",earthquake,,1.5,,,reviewed,ak,ak
2023-01-26T05:29:43.518Z,62.1501,-148.6362,34.3,1.2,ml,,,,0.35,ak,ak0231734hzj,2023-01-26T05:32:15.036Z,"40 km NNW of Chickaloon, Alaska",earthquake,,0.3,,,automatic,ak,ak
2023-01-26T05:21:19.220Z,33.4838333,-116.4046667,8.71,0.86,ml,25,90,0.1286,0.18,ci,ci40161943,2023-01-26T05:24:56.660Z,"22km SSW of La Quinta, CA",earthquake,0.26,1,0.113,21,automatic,ci,ci
2023-01-26T04:56:36.558Z,61.8635,-149.3468,19.8,1.5,ml,,,,0.55,ak,ak023172ou6f,2023-01-26T04:58:46.865Z,"14 km NNW of Fishhook, Alaska",earthquake,,0.2,,,automatic,ak,ak
2023-01-26T04:54:04.310Z,34.5181667,-118.6006667,8.67,1.33,ml,38,46,0.02513,0.25,ci,ci40161935,2023-01-26T05:04:54.920Z,"4km NNE of Castaic, CA",earthquake,0.33,0.95,0.12,28,automatic,ci,ci
2023-01-26T04:51:29.140Z,33.9033333,-118.7026667,15.68,1.75,ml,24,166,0.1329,0.28,ci,ci40161927,2023-01-26T04:55:10.919Z,"14km S of Malibu Beach, CA",earthquake,0.66,0.87,0.163,29,automatic,ci,ci
2023-01-26T04:46:25.230Z,33.9846667,-116.6858333,10.97,0.61,ml,11,65,0.06226,0.14,ci,ci40161919,2023-01-26T04:49:54.227Z,"12km SW of Morongo Valley, CA",earthquake,0.28,0.73,0.108,14,automatic,ci,ci
2023-01-26T04:43:50.800Z,33.4371667,-118.2303333,13.34,1.91,ml,7,212,0.4759,0.26,ci,ci40161911,2023-01-26T04:47:29.486Z,"14km NE of Avalon, CA",earthquake,2.39,31.61,0.197,25,automatic,ci,ci
2023-01-26T04:36:28.760Z,38.8268318,-122.8251648,0.93,1.9,md,10,101,0.008553,0.03,nc,nc73837511,2023-01-26T05:00:13.911Z,"8km NW of The Geysers, CA",earthquake,0.31,0.76,0.59,3,automatic,nc,nc
2023-01-26T04:32:15.710Z,18.9818325042725,-155.432495117188,36.5400009155273,2.12,ml,43,259,,0.140000001,hv,hv73313027,2023-01-26T04:37:46.490Z,"18 km ESE of Naalehu, Hawaii",earthquake,0.64,1.32000005,3.85,3,automatic,hv,hv
2023-01-26T04:29:44.721Z,-0.0988,-77.4813,10,4.1,mb,12,117,0.28,0.78,us,us6000jjbm,2023-01-26T05:03:59.040Z,"Ecuador",earthquake,1.95,1.975,0.261,4,reviewed,us,us
2023-01-26T04:23:51.367Z,2.8727,126.967,35,5.7,mww,90,38,2.125,0.82,us,us6000jjbj,2023-01-26T05:30:58.761Z,"171 km NW of Tobelo, Indonesia",earthquake,4.97,1.892,0.061,26,reviewed,us,us
2023-01-26T04:20:14.820Z,38.3904991,-122.4189987,5.77,1.6,md,12,137,0.04052,0.07,nc,nc73837506,2023-01-26T04:37:15.758Z,"5km WSW of Yountville, CA",earthquake,0.51,0.85,0.23,11,automatic,nc,nc
2023-01-26T04:16:06.492Z,16.6106,-25.6047,10,5.3,mb,58,72,9.064,0.68,us,us6000jjbh,2023-01-26T05:34:28.086Z,"73 km SW of Porto Novo, Cabo Verde",earthquake,10.44,1.864,0.026,488,reviewed,us,us
2023-01-26T04:07:56.356Z,-30.3135,-178.7238,35,4.4,mb,12,177,1.259,0.77,us,us6000jjbd,2023-01-26T04:48:51.040Z,"Kermadec Islands, New Zealand",earthquake,9.76,2.007,0.203,11,reviewed,us,us
2023-01-26T04:01:41.250Z,38.8371658,-122.8013306,2.24,0.78,md,9,67,0.00867,0.01,nc,nc73837501,2023-01-26T04:21:13.652Z,"7km WNW of Cobb, CA",earthquake,0.44,1.29,0.35,9,automatic,nc,nc
2023-01-26T03:43:15.465Z,61.8364,-150.8749,32,1.6,ml,,,,0.75,ak,ak0231720kmy,2023-01-26T03:45:19.850Z,"32 km ESE of Skwentna, Alaska",earthquake,,1.6,,,automatic,ak,ak
2023-01-26T03:35:13.550Z,19.1781673431396,-155.346832275391,27.8099994659424,1.89999998,md,41,184,,0.140000001,hv,hv73313012,2023-01-26T03:38:18.530Z,"14 km E of P?hala, Hawaii",earthquake,0.58,1.01999998,0.889999986,15,automatic,hv,hv
2023-01-26T03:16:22.081Z,-1.963,-79.9953,75.211,4.4,mb,21,137,0.491,0.29,us,us6000jjb9,2023-01-26T03:32:59.040Z,"4 km SW of El Triunfo, Ecuador",earthquake,5.37,9.012,0.204,7,reviewed,us,us
2023-01-26T02:26:19.140Z,17.9728333333333,-66.8753333333333,16.03,2.2,md,6,199,0.004339,0.07,pr,pr71394478,2023-01-26T02:38:48.370Z,"1 km ESE of Maria Antonia, Puerto Rico",earthquake,0.44,0.55,0.0628114320303155,5,reviewed,pr,pr
2023-01-26T02:23:06.320Z,33.1445,-116.501,11.35,0.89,ml,27,51,0.1029,0.21,ci,ci40161903,2023-01-26T02:26:38.486Z,"12km NE of Julian, CA",earthquake,0.34,0.76,0.102,14,automatic,ci,ci
